habitat-lab/habitat/datasets/rearrange/configs/hssd_ycb_multi_goal.yaml

```diff
@@ -0,0 +1,158 @@
+---
+    dataset_path: "data/fpss/hssd-hab-uncluttered.scene_dataset_config.json"
```

Comment:
FYI, this is the internal version for now and maybe should point to hssd public version instead.